client/Annotations/index.js

```diff
@@ -125,12 +125,19 @@ Template.Annotations.events({
                     });
                 } else {
                     AnnotationFiles.insert({"filename": fileName, "info": info, "annotations": csvAnnotations});
+                    filesSuccessfullyUploaded++;
+                    uploadsEnded++;
+                    if (uploadsEnded === allFiles.length) {
+                      loading.set(false);
+                      window.alert(`${allFiles.length - filesSuccessfullyUploaded}/${allFiles.length} files failed to upload:\n${filesUploadFailed}\n\n${filesSuccessfullyUploaded}/${allFiles.length} files successfully uploaded:\n${filesSuccessfullyUploadedString}`);
```

Comment:
filesUploadFailed is never used anywhere else in the code, so that'll always print an empty string in the pop-up. You should probably find a way to so some sort of error checking with AnnotationFiles' insert method so that the user can actually see what files failed to upload. The same holds with filesSuccessfullyUploadedString.

Reply:
Added the suggested changes. Unable to test whether upload failure works because I cannot find a way to fail uploading a file.